CONTRIBUTING.md

```diff
@@ -1,4 +1,10 @@
-# 💥 How to Contribute
+# 💥 Prerequisite Skills to Contribute.
+- [Git](https://git-scm.com/)
+- [node.js](https://nodejs.org/en/)
```

Comment:
@emmalearnscode Because when people want contribute file like out of docs then they always required a little bit knowledge about node.js, react also. They have to install all required dependencies so it's not so beginner task to do it. that's why I added.

Reply:
@shivam-sharma7 I also don't think we specifically need knowledge of Node.js to contribute to this project.

Reply:
what should I do? @emmalearnscode and @Shivansh-yadav13

Reply:
I think you should remove node.js as you don't need to know it to contribute. Prerequisites to contribute your profile: json. Prerequisites to contribute to documentation: markdown. To contribute components/css: React and Prime React.

Reply:
We can replace Nodejs with `npm` or `yarn`. What your thoughts? :thinking:

Reply:
@emmalearnscode @vikasganiga05 done